packages/fx-core/src/plugins/resource/sql/config.ts

```diff
@@ -20,4 +20,5 @@ export class SqlConfig {
   identity = "";
   existSql = false;
   skipAddingUser = false;
+  retryAddUser = 0;
```

Comment:
This is not the retry time configuration right? If not, we should move it to the logic code part. Seems to be a temp variable.

Reply:
It is fixed. Thanks.